Source/JavaScriptCore/wasm/WasmFunctionParser.h

```diff
@@ -3452,8 +3452,11 @@ FOR_EACH_WASM_MEMORY_STORE_OP(CREATE_CASE)
                 return { };
             }
 
+            if (exceptionLabel >= m_controlStack.size()) {
```

Comment:
Is there a reason not to use `WASM_PARSER_FAIL_IF` here? I think we should be able to pass it the same error message and condition.

Reply:
Yes, because this code is inside a lambda.  This is the way that the lambda returns an error.  See adjacent code.